packages/editor/src/components/post-actions/actions.js

```diff
@@ -307,7 +307,7 @@ const trashPostAction = {
 	},
 };
 
-function useTrashPostAction( postType ) {
+function useCanUserEligibilityCheckPostType( postType, capability, action ) {
 	const registry = useRegistry();
 	const { resource, cachedCanUserResolvers } = useSelect(
 		( select ) => {
```

Comment:
Could we avoid creating this subscription many times, and just do it once?

Reply:
Hi @ellatrix, I applied a change and now this PR does not add any store subscription.